cmd/webhook.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2016 Skippbox, Ltd.
```

Comment:
For new files, can we please use Copyright 2018 Bitnami instead, please? Thanks! (at some point we should do a pr for the rest)

Reply:
Makes sense.